main.star

```diff
@@ -241,9 +241,7 @@ def run(plan, args={}):
     if mev_endpoints:
         for index, participant in enumerate(all_participants):
             if args_with_right_defaults.participants[index].validator_count != 0:
-                mev_boost_launcher = mev_boost.new_mev_boost_launcher(
-                    MEV_BOOST_SHOULD_CHECK_RELAY, mev_endpoints
```

Comment:
lest delete the constant if it isn't being used?

Reply:
yes will do